ui/apps/platform/src/routePaths.ts

```diff
@@ -36,26 +46,23 @@ export const deprecatedPoliciesBasePath = `${mainPath}/policies`;
 export const deprecatedPoliciesPath = `${deprecatedPoliciesBasePath}/:policyId?/:command?`;
 export const riskBasePath = `${mainPath}/risk`;
 export const riskPath = `${riskBasePath}/:deploymentId?`;
-export const secretsPath = `${mainPath}/configmanagement/secrets/:secretId?`;
 export const searchPath = `${mainPath}/search`;
-export const apidocsPath = `${mainPath}/apidocs`;
-export const accessControlBasePath = `${mainPath}/access-control`;
-export const accessControlPath = `${accessControlBasePath}/:entitySegment?/:entityId?`;
-export const userBasePath = `${mainPath}/user`;
-export const userRolePath = `${userBasePath}/roles/:roleName`;
+export const secretsPath = `${mainPath}/configmanagement/secrets/:secretId?`;
 export const systemConfigPath = `${mainPath}/systemconfig`;
-export const complianceBasePath = `${mainPath}/compliance`;
-export const compliancePath = `${mainPath}/:context(compliance)`;
-export const complianceEnhancedBasePath = `${mainPath}/compliance-enhanced`;
-export const dataRetentionPath = `${mainPath}/retention`;
 export const systemHealthPath = `${mainPath}/system-health`;
-export const collectionsBasePath = `${mainPath}/collections`;
-export const collectionsPath = `${mainPath}/collections/:collectionId?`;
-export const listeningEndpointsBasePath = `${mainPath}/listening-endpoints`;
+export const userBasePath = `${mainPath}/user`;
+export const userRolePath = `${userBasePath}/roles/:roleName`;
+export const violationsBasePath = `${mainPath}/violations`;
+export const violationsPath = `${violationsBasePath}/:alertId?`;
+export const vulnManagementPath = `${mainPath}/vulnerability-management`;
+export const vulnManagementReportsPath = `${vulnManagementPath}/reports`;
+export const vulnManagementRiskAcceptancePath = `${vulnManagementPath}/risk-acceptance`;
+export const vulnerabilitiesBasePath = `${mainPath}/vulnerabilities`;
+export const vulnerabilitiesWorkloadCvesPath = `${vulnerabilitiesBasePath}/workload-cves`;
+export const vulnerabilityReportsPath = `${vulnerabilitiesBasePath}/reports`;
 
 // Configuration Management
 
-export const configManagementPath = `${mainPath}/configmanagement`;
 export const configManagementClustersPath = `${configManagementPath}/clusters`;
 export const configManagementControlsPath = `${configManagementPath}/controls`;
 export const configManagementDeploymentsPath = `${configManagementPath}/deployments`;
```

Comment:
Are the paths here and below that were not moved into the alphabetized list ones that are used only for routing inside the Container level? If so, I think it would be helpful noting that in comments that split the two sections so that new top level routes are sure to be added to the list above.

Reply:
Thank you for asking. A possible solution is move them from global to container scope.